CanDIG/CanDIGv2
# Sonchau/keycloak

## What's new
- Upgrade keycloak from 16 to 24
- Update Tyk logout with id_token when logout
- Clean up

#### How to test
- Pull this branch and rebuild
- Run test-integration
- Login portal and logout, should redirect to the login page
